spec/models/infrastructure_spec.rb

```diff
@@ -5,6 +5,9 @@
     let(:infrastructure) { create :infrastructure }
     let(:cluster_template) { create :cluster_template }
     before(:each) do
+      %w(yggdrasil consul kibana kafka zookeeper elasticsearch barito-flow-consumer barito-flow-producer).each do |n|
```

Comment:
we should make this as spec seeds

Reply:
Done https://github.com/BaritoLog/BaritoMarket/commit/df772f2e655b3fee57db1553827078851fc4255f